.gitignore

```diff
@@ -27,6 +27,8 @@ build/
 .idea
 .gradle
 local.properties
+espeogoogleplay.json
+espeo-key.keystore
```

Comment:
i think we have general rule to disable all keystores which has not debug in name. So this line in not needed.

Reply:
We alraedy have !debug.keystore